modules/social_features/social_event/src/Form/EventSettingsForm.php

```diff
@@ -33,14 +33,14 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $group_type_ids = $this->configFactory->getEditable('social_event.settings')
-      ->get('enroll');
+    // Get the config.
+    $social_event_config = $this->configFactory->getEditable('social_event.settings');
```

Comment:
We don't need `getEditable()` here, we're not editing anything right? We can use `get()`.

Reply:
In forms we should always use `getEditable` otherwise config overrides might accidentally get moved into database configuration because it'll be put into the form.

🏎 💨